scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt

```diff
@@ -332,6 +335,13 @@ internal class ScenarioServiceImpl(
       this.findAllScenariosStateOption(organizationId, workspaceId, true)
           .addLastRunsInfo(this, organizationId, workspaceId)
 
+  override fun findAllScenariosByValidationStatus(
+      organizationId: String,
+      workspaceId: String,
+      validationStatus: ScenarioValidationStatus
+  ): List<Scenario> =
+      findAllScenarioByValidationStatus(organizationId, workspaceId, validationStatus.toString())
```

Comment:
Could be interesting to use `validationStatus.value` instead of `validationStatus.toString()`.
toString method can be overridden